partner_contact_birthdate/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright 2014-2015  Grupo ESOC <www.grupoesoc.es>
```

Comment:
Remove copyright lines from init files.

Reply:
@NL66278 
Please suggest me that why init files don't have copyright lines but other files such as res_partner.py have copyright lines.

Thank you.

Reply:
Actually the contributing guidelines are not speaking anymore about this. But the logic is that there is not really anything to claim copyright for...

Reply:
@NL66278 
I got it. This is new for me.

Thank you for reviewed :)